### ISSUE TITLE
README UpToDate maken

### ISSUE THREAD

**Issue by JesseZwitserlood** [AmpersandTarski/RAP#135]
Currently there are broken links in the readme file, these need do be updated to make sure it works as a introduction and a 'to do before progressing' file

- [ ] README is uptodate

**Comment by stefjoosten** [comment]
Is there anything we can do to prevent broken links systematically?

**Comment by hanjoosten** [comment]
It is unclear to me which README we are talking about here. Is it [this one](https://github.com/AmpersandTarski/RAP/blob/master/README.md)?

**Comment by hanjoosten** [comment]
Just an idea: If we would restrict our README files only to a single link to our documentation, we can narrow the scope of broken links to broken links in our documentation. Docusaurus helps with this, because it checks for broken links by default.

**Comment by Michiel-s** [comment]
> Is there anything we can do to prevent broken links systematically?

Yes, that is a setting in Docusaurus. We can opt-in for that  when everything works. The build will fail with broken links, so this is not yet the moment I think.

**Comment by hanjoosten** [comment]
> > Is there anything we can do to prevent broken links systematically?
> 
> Yes, that is a setting in Docusaurus. We can opt-in for that when everything works. The build will fail with broken links, so this is not yet the moment I think.

True. Currently I am combining pages to get far less documents and much longer pages. It breaks links all over the place, so it is convenient that broken links are (temporarily) acceptable.
